ninja.py

```diff
@@ -0,0 +1,231 @@
+# FRUIT NINJA #
+# MP4: INTERACTIVE PROGRAMMING #
+# AVA LAKMAZAHERI AND EMMA WESTERHOFF #
+# SOFTWARE DESIGN FALL 2017 #
+
+import random, pygame, sys, argparse, imutils, cv2, copy, math
+import numpy as np
+import inspect
+from pygame.locals import *
+from all_fruits import *
+from Scoring import *
+
+# Hand Tracking Parameters
```

Comment:
All of the functions in this file should be in a 'game' class. Having it freeform like this makes me sad.